### DISCUSSION TITLE
/cmi spawnmob pig|strider with saddle?

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#4269] [Feature Requests]
Is it possible to force the 1.16.3 engine to spawn a mob like pig and strider with their saddle?

```
/cmi spawnmob pig:adult:saddle
/cmi spawnmob strider:saddle
```

**Comment by CloudeLecaw** [comment]
This also would be nice for horse, doney. mule, tradelamer and lama (there its a carpet)

```
/cmi spawnmob horse:adult:saddle
/cmi spawnmob mule:adult:saddle
/cmi spawnmob donkey:adult:saddle
/cmi spawnmob llama:adult:saddle:color
/cmi spawnmob trader_llama:adult:saddle:color
```

**Comment by mrfloris** [comment]
It's 2023, and we have strider and camel and stuff, maybe if it is technically possible to do and not much of a hassle, <3 could we?

https://minecraft.fandom.com/wiki/Saddle

camels, horses, donkeys, mules and skeleton horses, pigs and striders,

**Reply by CloudeLecaw** [reply]
Would be wonderfull <3

**Comment by mrfloris** [comment]
Colored leather is already possible; but not yet rgp, next update!

`/spawnmob zombie:boots-leatherboots/123/255/23`
`/spawnmob zombie:boots-leatherboots/red`

<img width="415" alt="Screen-Shot 2023-10-11 at 16 26 20" src="https://github.com/Zrips/CMI/assets/28841349/719f725c-c76a-4aa5-ab37-013c4696fca4">
<img width="474" alt="Screen-Shot 2023-10-11 at 16 26 16" src="https://github.com/Zrips/CMI/assets/28841349/ab780a95-31ad-4d1f-b01b-afed553d938d">

**Comment by CloudeLecaw** [comment]
wasnt this added in last version?
